.concourse/kubecf.yaml

```diff
@@ -31,6 +31,7 @@ branches: # Repository branches to track
 - release-2.3
 pr_resources:
 - pr
+pr_base_branch: ~ # filter PRs depending on the branch they target. `~` for all branches
```

Comment:
based on the comment below, the null value here doesn't mean "all branches", so this comment is wrong?

Reply:
the `# filter PRs depending on the branch they target. ~ for all branches` is a comment of the option `pr_base_branch`. The implementation is different.

I prefer to have something like `~` to denote "test all branches", rather than using `all` or `all-branches` or whatever. With our luck, we would get a branch named `all` to make a mess of it.

Reply:
Another option would be to act if the `pr_base_branch` key is present, and leave it commented in the docs. But people tend to overlook commented code.